internal/opa/rule/rule.go

```diff
@@ -219,20 +219,28 @@ func documentationUrl(a *ast.AnnotationsRef) string {
 		return ""
 	}
 
-	// Notes:
-	// - This makes the assumption that we're looking at our own Conforma rules with
-	//   docs in the enterprise-contract github pages. That's not likely to be true
-	//   always. A future improvement for this might include a way to extract a
-	//   docs url from a package annotation instead using the hard-coded url here.
-	// - The length test is because we're expecting pathStrings to be like this:
-	//     data.policy.release.some_package_name.deny
-	//   Avoid errors indexing pathStrings and also try to avoid showing a url
-	//   if it's unlikely to be a real link to existing docs.
-	ruleDocUrlFormat := "https://conforma.dev/docs/ec-policies/%s_policy.html#%s__%s"
+	// This makes assumptions about the way we publish policy docs for
```

Comment:
Removing this entirely and bringing back more functional version in future  is a reasonable suggestion. It's definitely under-cooked.

I'm open to suggestions, but my thinking is that if we clearly describe what it's flaws are, it will be easier to fix it later.